packages/react-native/src/NotifeeApiModule.ts

```diff
@@ -672,27 +774,27 @@ export default class NotifeeApiModule extends NotifeeNativeModule implements Mod
   };
 
   public isBatteryOptimizationEnabled = (): Promise<boolean> => {
-    if (isIOS) {
+    if (!isAndroid) {
       return Promise.resolve(false);
     }
 
     return this.native.isBatteryOptimizationEnabled();
   };
 
   public openBatteryOptimizationSettings = (): Promise<void> => {
-    if (isIOS) {
+    if (!isAndroid) {
       return Promise.resolve();
     }
     return this.native.openBatteryOptimizationSettings();
   };
 
   public getPowerManagerInfo = (): Promise<PowerManagerInfo> => {
-    if (isIOS) {
-      // iOS doesn't support this, so instead we
+    if (!isAndroid) {
+      // only Android supports this, so instead we
       // return a dummy response to allow the power manager
-      // flow work the same on both iOS & Android
+      // flow work the same on all platforms
       return Promise.resolve({
-        manufacturer: 'apple',
```

Comment:
@helenaford this is the only line that is not 100% backwards compatible. I think we should just put the platform in here so I implemented it as such, otherwise if we want 100% backwards compatibility on the return value (that no one is probably using?) we need to special case Platform.OS === ios to do it

Reply:
@mikehardy mmm interesting, yeah i looked and it's for the  `getPowerManagerInfo` endpoint... probably OK like you say 😅 

I was slightly nervous seeing this as I have a big flutter p/r coming, but the changes look good. Only changes under `packages/react-native` 😎 nice!

Reply:
Yep - adding react-native-web support is actually really easy. If I tried to post my tvOS one again that might perturb things, no plans for that at the moment.... Flutter PR, nice!